### ISSUE TITLE
Wrong compilation paths for custom themes

### ISSUE THREAD

**Issue by Igloczek** [SnowdogApps/magento2-frontools#8]
Currently we keep source files directory tree in styles compilation process and this is not so good, because we should remove `web` folder from the middle of destination path.

Current:
`vendor/theme/web/css/custom.scss` -> `vendor/theme/locale/web/css/custom.css`

Correct:
`vendor/theme/web/css/custom.scss` -> `vendor/theme/locale/css/custom.css`

**Comment by redrohX** [comment]
Was this causing the compiled css files to appear in the css/css directory? I updated yesterday to 0.4.1 and this happened for me. I rolled back to 0.4.0 this morning and it works without issues.

**Comment by Igloczek** [comment]
Thanks for report @redrohX Could you write smth more about source and output files structure? Also happens on custom theme or you use default?

**Comment by redrohX** [comment]
It happened on all themes (blank and luma), including my own custom theme that extends blank.
in themes.json I configured my own theme like this. Which is a similar config to blank and luma:
```js
"default": {
    "src": "../app/design/frontend/kega/default",
    "dest": "../pub/static/frontend/kega/default",
    [ .... ]
    "files": [
      "css/styles-m",
      "css/styles-l"
    ]
  },
```

**Comment by Igloczek** [comment]
Did you use `default: true` in config? If not, you should.

**Comment by redrohX** [comment]
Yes I did. Everything the same as luma, except for the paths, vendor and name.

**Comment by Igloczek** [comment]
So I definitely need to dig into this :)

**Comment by Igloczek** [comment]
Resolved in 0.5.0
